src/Core/Configurations/ConfigFileWatcher.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO.Abstractions;
+using Azure.DataApiBuilder.Config;
+using Azure.DataApiBuilder.Config.ObjectModel;
+using Path = System.IO.Path;
+
+namespace Azure.DataApiBuilder.Core.Configurations;
+
+public class ConfigFileWatcher
+{
+    private FileSystemWatcher? _fileWatcher;
+    RuntimeConfigProvider? _configProvider;
+
+    public ConfigFileWatcher(RuntimeConfigProvider configProvider)
+    {
+        FileSystemRuntimeConfigLoader loader = (FileSystemRuntimeConfigLoader)configProvider.ConfigLoader;
+        string configFileName = loader.ConfigFilePath;
```

Comment:
it is, I was just used to only seeing the name when running locally

Reply:
it can either be a simple file name in the case where the config is already in the current directory you can just supply the name, or it can be a complete path. I will update the code to handle both cases correctly.